src/utils/index.js

```diff
@@ -2,7 +2,8 @@
 import Form from 'mobx-react-form';
 
 export function parseDateTime(timeNumber: Number) {
-  const date: Date = new Date(timeNumber);
+  // convert epoch timestamp to miliseconds
+  const date: Date = new Date(timeNumber * 1000);
```

Comment:
momentjs ?

Reply:
no momentjs dependency since there is only 1 function needed for the use of date time re-format